optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/valuerange/descriptor/EntityIndependentValueRangeDescriptor.java

```diff
@@ -25,10 +25,10 @@
 public interface EntityIndependentValueRangeDescriptor<Solution_> extends ValueRangeDescriptor<Solution_> {
 
     /**
-     * As specified by {@link #extractValueRange(Solution_, Object)}.
+     * As specified by {@link ValueRangeDescriptor#extractValueRange}.
```

Comment:
I don't understand: why is it ok to leave out the parameters here, but not in all the other cases with Solution_? What's the difference?